src/tests/src/libs/antares/study/short-term-storage-input/short-term-storage-input-output.cpp

```diff
@@ -451,3 +455,478 @@ BOOST_FIXTURE_TEST_CASE(check_series_save, Fixture)
 }
 
 BOOST_AUTO_TEST_SUITE_END()
+
+BOOST_AUTO_TEST_SUITE(AdditionalConstraintsTests)
+
+BOOST_AUTO_TEST_CASE(Validate_ClusterIdEmpty)
```

Comment:
we should not hesitate to make test title look like a sentence :  
**Validate_ClusterIdEmpty** ==> **cluster_id_is_empty___we_get_validation_error**